kuma/static/js/wiki.js

```diff
@@ -883,4 +883,17 @@
         }
     }
 
+    // listens for post message from interactive editor
+    window.addEventListener('message', function(event) {
+        // get the interactive editor iframe
+        var iframe = document.querySelector('iframe.interactive');
```

Comment:
Indeed. Currently we can be confident that there will only be one iframe with the class `interactive`, even if there are two somehow, the one I am targeting will most definitely be the first in the DOM so, this will grab that one.

With all of that said though, as we will currently(and for the immediate future) only have one per page, perhaps it is worth adding an id to the iframe? Something like `id="interactive-editor"`